testing/functional/helpers/scheduler.test.helper.ts

```diff
@@ -11,6 +11,30 @@ export default class SchedulerTestHelper {
         return this.scheduler.find(`.dx-scheduler-appointment`).nth(index);
     }
 
+    getAppointmentByTitle(title) {
```

Comment:
I would suggest adding the second parameter `index`, which would be in `nth` statement

Reply:
Yes. This is a very good point.